new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/AppBuilder.java

```diff
@@ -146,6 +179,32 @@ public boolean buildZipArchive(
         .execute();
   }
 
+  /**
+   * Build a zip archive that includes coh-proxy-server.gar in the srcDir.
+   *
+   * @param zipPath zip file path for the resulting archive
+   * @param srcDir source directory
+   */
+  public boolean buildCoherenceZipArchive(String zipPath, String srcDir) {
```

Comment:
Can you check if the existing method to build the archive be used?

Reply:
I did. existing method create a .ear/war fiule and then zip. what we need for this test is .gar and then zip

Reply:
This method is used to build archive for coherence-proxy-server. After discussing with Bhavani, we both agree to keep it as it is